.github/workflows/sync_crowdin.yml

```diff
@@ -26,10 +28,10 @@ jobs:
           download_translations: true
           localization_branch_name: feat/translations
           create_pull_request: true
-          pull_request_title: "chore(i18n): Sync translations"
-          pull_request_body: "Sync translations from [Crowdin/ReVanced](https://crowdin.com/project/revanced)"
+          pull_request_title: "chore: Sync translations"
+          pull_request_body: "Sync translations from [crowdin.com/project/revanced](https://crowdin.com/project/revanced)"
           pull_request_base_branch_name: "dev"
-          commit_message: "chore(i18n): Sync translations"
```

Comment:
We do not have any particular purpose for these scopes yet, keeping it simple by removing them

Reply:
Why remove them, they can stay for consistency with other commits

Reply:
They are removed for consistency with other commits. We do not have any use for scopes. Currently, they are used randomly without any convention.